src/@types/crypto.ts

```diff
@@ -30,7 +30,7 @@ interface Extensible {
 
 /* eslint-disable camelcase */
 
-/** The result of a call to {@link MatrixClient.exportRoomKeys} */
+/** The result of a call to CryptoApi.exportRoomKeys */
```

Comment:
This looks like it breaks docs

Reply:
See https://github.com/matrix-org/matrix-js-sdk/pull/4672

Reply:
![image](https://github.com/user-attachments/assets/37833035-84fc-4aa3-a2dd-2cf8c2ec0b75)

It works as per the docs I linked you to when you asked

Reply:
Oh perfect, I didn't understood first the documentation. I tried multiple times different without success. I'll make a PR for that

Reply:
Also just noticed looks like `OlmGroupSessionExtraData` is now unused that `/crypto/` has been deleted so seems like the checks for dead code were not exhaustive

Reply:
I relied on knip to catch the unused type, this one wasn't raised unfortunately.